packages/tab/index.js

```diff
@@ -113,6 +130,12 @@ export default class Tab extends Component {
         role='tab'
         aria-selected={ariaSelected}
         tabIndex={tabIndex}
+        onClick={(e) => {
+          if (onClick) {
+            const tabEvent = new e.constructor(e.type, e, {detail: {tab: this}});
```

Comment:
what is happening here?, you cannot pass the evt? i dont think MDCW is doing this so why do we?

Reply:
`MDCTabBarFoundation.handleTabInteraction` requires that the tab be passed into the `event.detail`, MDCW does this in the custom `MDCTab:interacted` event

Reply:
I think MDCW shouldn't be using the tab in the foundation code. Its strange that we need to construct an event in this Tab component for MDC Tab Bar to consume. 

I like your idea of using id's like with chips

Reply:
Talked offline, sounds good. We can call `foundation.activateTab` directly with the tab's index (instead of assigning an id) and bypass `foundation.handleTabInteraction` entirely